go.mod

```diff
@@ -1,30 +1,92 @@
 module knative.dev/test-infra
 
-go 1.15
+go 1.18
 
 require (
-	cloud.google.com/go/iam v0.1.0 // indirect
-	cloud.google.com/go/storage v1.10.0
+	cloud.google.com/go/storage v1.22.1
 	github.com/blang/semver/v4 v4.0.0
 	github.com/davecgh/go-spew v1.1.1
-	github.com/go-git/go-git-fixtures/v4 v4.0.1
-	github.com/go-git/go-git/v5 v5.1.0
+	github.com/go-git/go-git-fixtures/v4 v4.2.1
+	github.com/go-git/go-git/v5 v5.4.2
 	github.com/go-sql-driver/mysql v1.5.0
-	github.com/google/go-cmp v0.5.7
-	github.com/google/go-containerregistry v0.8.0
+	github.com/google/go-cmp v0.5.8
+	github.com/google/go-containerregistry v0.8.1-0.20220216220642-00c59d91847c
 	github.com/google/go-github/v32 v32.1.1-0.20201004213705-76c3c3d7c6e7 // HEAD as of Nov 6
 	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/pkg/errors v0.9.1
-	github.com/spf13/cobra v1.3.0
-	go.uber.org/atomic v1.7.0
-	golang.org/x/mod v0.5.1
-	golang.org/x/net v0.0.0-20220127200216-cd36cc0744dd
-	golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
-	google.golang.org/api v0.70.0
-	google.golang.org/genproto v0.0.0-20220222213610-43724f9ea8cf // indirect
-	k8s.io/apimachinery v0.20.6
-	knative.dev/hack v0.0.0-20220823140917-8d1e4ccf9dc3
-	sigs.k8s.io/boskos v0.0.0-20200729174948-794df80db9c9
-	sigs.k8s.io/yaml v1.2.0
+	github.com/spf13/cobra v1.4.0
+	go.uber.org/atomic v1.9.0
+	golang.org/x/mod v0.6.0-dev.0.20220419223038-86c51ed26bb4
+	golang.org/x/net v0.0.0-20220607020251-c690dde0001d
+	golang.org/x/oauth2 v0.0.0-20220608161450-d0670ef3b1eb
+	google.golang.org/api v0.83.0
+	k8s.io/apimachinery v0.24.2
+	knative.dev/hack v0.0.0-20220902140220-97fd663bf52b
+	sigs.k8s.io/boskos v0.0.0-20210730172138-093b54882439
+	sigs.k8s.io/yaml v1.3.0
+)
+
+// TODO: remove this once we have a fixed knative-hack (https://github.com/knative/hack/issues/216)
+replace knative.dev/hack => github.com/cardil/knative-hack v0.0.0-20220902210110-5d40dd4a1f82
```

Comment:
I think this one was already merged? Let's remove the TODO

Reply:
Yes, this should be removed.